### ISSUE TITLE
[REPORT]: Some CHT are just written wrong, I modify them myself, the ones that don't work

### ISSUE THREAD

**Issue by 666PS2** [PS2-Widescreen/OPL-Widescreen-Cheats#77]
### Checks

- [X] I have checked if a report of the same game, in the same test environment, is already included
- [X] I have tested this __on real hardware__ either with OpenPS2Loader or running a phisical copy of the game
- [X] I have confirmed that __NONE__ of the mentioned issues in this report ocurr if the game is ran without the hack

### gameplay

barely working

### Console model

SCPH-90010 tbm um SCPH-75001

### Version of the software used to run the cheats

OPNPS2LD-v1.2.0-Beta-2109-90cb644.

### title

there are some games

### Game Region code (ID)

blablabla

### device used on test

SMB

### Explain your issue

In some games, the widscreen CHT doesn't work because the code is wrong or some code is getting in the way of the list you made. There are some CHTs with problems! In the wrong writing or code, Google translate's fault if something is written wrong.

**Comment by israpps** [comment]
what do you mean by written wrong?????

At least give an example so I can check them out

**Comment by 666PS2** [comment]
> o que você quer dizer com escrito errado?????
> 
> Pelo menos dê um exemplo para que eu possa verificar

an example is Dragon Ball Z Budokai 3 (Greatest Hits) (USA) his cht doesn't work the widscreen of the one on your list I just downloaded another one from the internet and got the code and pasted it on your cht then yes it worked widthscreen

**Comment by israpps** [comment]
> > o que você quer dizer com escrito errado?????
> > Pelo menos dê um exemplo para que eu possa verificar
> 
> an example is Dragon Ball Z Budokai 3 (Greatest Hits) (USA) his cht doesn't work the widscreen of the one on your list I just downloaded another one from the internet and got the code and pasted it on your cht then yes it worked widthscreen

Yeah, nothing I can do with vague reports.

Maybe you have a modified version of the game and that makes the master code fail?

**Comment by GhostUser6** [comment]
> > > o que você quer dizer com escrito errado????? 
> > > Pelo menos dê um exemplo para que eu possa verificar
> > 
> > 
> > um exemplo é Dragon Ball Z Budokai 3 (Greatest Hits) (EUA) o cht dele não funciona na tela larga do que está na sua lista acabei de baixar outro da internet e peguei o código e colei no seu cht então sim funcionou na tela larga
> 
> Sim, não posso fazer nada com relatórios vagos.
> 
> Talvez você tenha uma versão modificada do jogo e isso faça o código mestre falhar?

yes maybe it's because it's the Dragon Ball Z Budokai 3 (Greatest Hits) (USA) SLUS-20998 version but it doesn't make sense because I simply got the code from a pnach file this code here 
204280B4 3F0AD706
2044AF20 3F0AD706
I changed your cht file and the widscreen worked

here is the file where I used the code
[C97EF0A4.pnach.txt](https://github.com/user-attachments/files/16732747/C97EF0A4.pnach.txt)

**Comment by GhostUser6** [comment]
> > > > o que você quer dizer com escrito errado????? 
> > > > Pelo menos dê um exemplo para que eu possa verificar
> > > 
> > > 
> > > um exemplo é Dragon Ball Z Budokai 3 (Greatest Hits) (EUA) o cht dele não funciona na tela larga do que está na sua lista acabei de baixar outro da internet e peguei o código e colei no seu cht então sim funcionou na tela larga
> > 
> > 
> > Sim, não posso fazer nada com relatórios vagos.
> > Talvez você tenha uma versão modificada do jogo e isso faça o código mestre falhar?
> 
> sim, talvez seja porque é a versão Dragon Ball Z Budokai 3 (Greatest Hits) (EUA) SLUS-20998, mas não faz sentido porque eu simplesmente peguei o código de um arquivo pnach, este código aqui 204280B4 3F0AD706 2044AF20 3F0AD706 Eu alterei seu arquivo cht e o widscreen funcionou
> 
> aqui está o arquivo onde usei o código [C97EF0A4.pnach.txt](https://github.com/user-attachments/files/16732747/C97EF0A4.pnach.txt)

Now I understand why it didn't work in this game because your cht is only for the normal version, not for Dragon Ball Z: Budokai 3 (Greatest Hits) as I got the version code (Greatest HIts) it worked, sorry for the confusion 

cht that doesn't work for me, I've modified some things and it works and it's very confusing for me because I don't have much experience but I'm getting something.

**Comment by Jaajees** [comment]
If the issue has already been resolved, you can close it.

**Comment by GhostUser6** [comment]
@israpps If you don't want anything else, could you tell me where you can find these codes?
